front/app/containers/Admin/projects/project/inputImporter/ImportModal/ImportPdfModal.tsx

```diff
@@ -54,6 +54,10 @@ const ImportPdfModal = ({ open, onClose, onImport }: Props) => {
     projectId: string;
   };
 
+  // TEMP: Remove this when the legacy PDF format is no longer required
+  const [searchParams] = useSearchParams();
+  const legacyPdfImport = searchParams.get('legacy') === 'true' || false;
```

Comment:
Could just be:
```suggestion
  const legacyPdfImport = searchParams.get('legacy') === 'true';
```

Reply:
Or no variable at all if we'd rename to `legacy_pdf`, then the advantage of the variable (making the param clearer) disappears.